src/main.rs

```diff
@@ -169,6 +238,31 @@ async fn main() -> anyhow::Result<()> {
     }
 
     info!("Done");
-
     Ok(())
 }
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let cli = Cli::parse();
+
+    let bucket_config = BucketConfig::from_env()?;
+
+    match &cli.command {
```

Comment:
You can remove the & here which would allow you to remove the `*`s below.

Reply:
I'll add that in the next PR